cppwinrt/code_writers.h

```diff
@@ -398,17 +416,18 @@ namespace cppwinrt
 
         if (empty(generics))
         {
-            auto format = R"(    template <> inline constexpr guid guid_v<%>{ % };
+            auto format = R"(    template <> inline constexpr guid guid_v<%>{ % }; // %
 )";
 
             w.write(format,
                 type,
-                bind<write_guid_value>(attribute.Value().FixedArgs()));
+                bind<write_guid_value>(attribute.Value().FixedArgs()),
```

Comment:
Thanks for the contribution! Can you hoist `attribute.Value().FixedArgs()` above the `if` statement and reuse the resulting `fixed_args` for these (4) calls to avoid repeated parsing.